authors/apps/authentication/tests_/test_signup_validation.py

```diff
@@ -0,0 +1,114 @@
+import json
+
+from django.urls import reverse
+from rest_framework import status
+
+from authors.apps.authentication.models import User
+
+from .test_base_class import BaseTestClass
+
+
+class TestSignupValidation(BaseTestClass):
+    """This class tests the functionality of the registration validation
+
+    Examples:
+    - Registration with a very short password
+    - Registration with non alphanumeric password
+    - Registration with an already existing email
+    
+    """
+
+    def test_signup_with_short_password(self):
```

Comment:
@d-rita following the test naming convention, this test should be 
```suggestion
    def test_signup_with_short_password_fails(self):
```
incase of a success  test:
```suggestion
    def test_signup_with_short_password_succeeds(self):
```
Please add the approphriate post_fixes to the test names

Reply:
Okay. Noted